image[] rgb_lists
image[] depth_lists
---
image[] gradient_lists
